core/videos/admin.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.contrib import admin
+from .models import Video
+# Register your models here.
+admin.site.register(Video)
```

Comment:
For the common conventions, there is always a newline character at the end of the file.